### PR TITLE
Add SogeCommerce endpoint

### PR DESCRIPTION
**Pull request by camilleislasse**
Add SogeCommerce endpoint

### CODE REVIEW

**Comment by camilleislasse** [comment]
@gplanchat

File: src/Form/Type/PayzenGatewayConfigurationType.php [hunk 1]
```diff
@@ -44,6 +44,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     'sylius.form.gateway_configuration.payzen.endpoint.choices.scellius' => Api::ENDPOINT_SCELLIUS,
                     'sylius.form.gateway_configuration.payzen.endpoint.choices.systempay' => Api::ENDPOINT_SYSTEMPAY,
                     'sylius.form.gateway_configuration.payzen.endpoint.choices.osb' => Api::ENDPOINT_OSB,
+                    'sylius.form.gateway_configuration.payzen.endpoint.choices.soge_commerce' => Api::ENDPOINT_SOGECOMMERCE,
```
Comment (gplanchat) [inline src/Form/Type/PayzenGatewayConfigurationType.php]:
```suggestion
                    'sylius.form.gateway_configuration.payzen.endpoint.choices.sogecommerce' => Api::ENDPOINT_SOGECOMMERCE,
```

File: src/Resources/translations/messages.en.yml [hunk 2]
```diff
@@ -17,5 +17,6 @@ sylius:
                         scellius: Scellius
                         systempay: SystemPay
                         osb: OSB
+                        soge_commerce: SogeCommerce
```
Comment (gplanchat) [inline src/Resources/translations/messages.en.yml]:
```suggestion
                        sogecommerce: SogeCommerce
```

File: src/Resources/translations/messages.fr.yml [hunk 3]
```diff
@@ -17,5 +17,6 @@ sylius:
                         scellius: Scellius
                         systempay: SystemPay
                         osb: OSB
+                        soge_commerce: SogeCommerce
```
Comment (gplanchat) [inline src/Resources/translations/messages.fr.yml]:
```suggestion
                        sogecommerce: SogeCommerce
```

File: README.md [hunk 4]
```diff
@@ -3,6 +3,14 @@ Sylius Payzen Bundle
...
 Sylius PayZen bundle via Payum
 
+### Compatibility
+
+- Systempay
+- Scellius
+- CLIC&PAY
+- OSB
+- SogeCommerce
+
```
Comment (gplanchat) [inline README.md:13]:
```suggestion
If the payment gateway you are using is not in this list, please open an [issue](https://github.com/kiboko-labs/payzen-bundle/issues) or a [pull request](https://github.com/kiboko-labs/payzen-bundle/pulls)
```

**Review by gplanchat — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -3,6 +3,14 @@ Sylius Payzen Bundle
...
 Sylius PayZen bundle via Payum
 
+### Compatibility
+
+- Systempay
+- Scellius
+- CLIC&PAY
+- OSB
+- SogeCommerce
+If the payment gateway you are using is not in this list, please open an [issue](https://github.com/kiboko-labs/payzen-bundle/issues) or a [pull request](https://github.com/kiboko-labs/payzen-bundle/pulls)
```
Comment (gplanchat) [inline README.md:14]:
```suggestion

If the payment gateway you are using is not in this list, please open an [issue](https://github.com/kiboko-labs/payzen-bundle/issues) or a [pull request](https://github.com/kiboko-labs/payzen-bundle/pulls)
```

**Review by gplanchat — APPROVED** [review]

**Review by gplanchat — COMMENTED** [review]
It is missing a composer requirement `"ekyna/payum-payzen-bundle": "^1.6.6"`

**Pull request merged**